source/features/hide-readme-header.css

```diff
@@ -22,7 +22,7 @@
 	right: 0;
 	top: 0;
 	opacity: 0.2;
-	transition: opacity 250ms;
+	transition: opacity 0.25s;
```

Comment:
y tho 😅

Reply:
Felt the irresistible itch to make the durations uniform 😅